lib/loggerator/loggerator.rb

```diff
@@ -11,7 +11,7 @@ def log(data, &block)
     Log.to_stream(Log.stdout, Log.contexts(data), &block)
   end
 
-  def log_error(e=$!, data = {})
+  def log_error(e=$ERROR_INFO, data = {})
```

Comment:
@reidmix Addressed this issue here and will submit a PR once this is merged.

https://github.com/heroku/loggerator/compare/jmervine-rails-refactor...jmervine-log_error-bug